### ISSUE TITLE
Influx DB mit Passwort

### ISSUE THREAD

**Issue by IGLO70** [matthewdowney/influxdb-stream#2]
Gibt es eine Möglichkeit, auch eine mit Benutzer und Passwort geschützte Datenbank mit diesem Tool zu Exportieren. Wie müsste der Benutzername und das Passwort übergeben werden?

**Comment by matthewdowney** [comment]
No, unfortunately not, you'd have to modify the code making the request to add authentication. 

https://github.com/matthewdowney/influxdb-stream/blob/5c31bb67390f56d145fb65650b6c4ee2a883c3c8/src/influxdb_stream/core.clj#L61

Happy to accept a PR for this!

**Comment by loxatsonnenberg** [comment]
Hi,
I added to the conf.edn file
`:port          8086
:u             "username"
:p             "password"
:db            "dbname"`

and modified the core.clj 
`  (let [resp (client/post
               (format "http://%s:%s/query?U=%s&p=%s" host port u p)`

would you please compile a new jar file, because I'm not familiar with that tool.

Thank you.

**Comment by matthewdowney** [comment]
@loxatsonnenberg just to confirm, the query params are called (uppercase) `U` and `p` like that?

**Comment by loxatsonnenberg** [comment]
thanks for that hint:
according [influxdb-manual](https://docs.influxdata.com/influxdb/v1.8/administration/authentication_and_authorization/#credentials-as-query-parameters) both are lowercase

**Comment by matthewdowney** [comment]
Ok, added support for `:username` and `:password` conf fields in a new branch. ([These](https://github.com/matthewdowney/influxdb-stream/compare/support-auth?diff=split) are the changes.)

Release with JAR file is [here](https://github.com/matthewdowney/influxdb-stream/releases/tag/v0.2.0-PRELIMINARY). Let me know if this works for you.

**Comment by matthewdowney** [comment]
(it should also probably go without saying that my advice is to ditch InfluxDB ASAP lol)

**Comment by matthewdowney** [comment]
@loxatsonnenberg are you able to confirm if this worked for you?
